netqasm/sdk/memmgr.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import annotations
+
+from itertools import count
+from typing import Dict, List, Set
+
+from netqasm.lang import operand
+from netqasm.lang.encoding import REG_INDEX_BITS, RegisterName
+from netqasm.lang.parsing.text import parse_register
+from netqasm.sdk.futures import Array
+from netqasm.sdk.qubit import Qubit
+
+
+class MemoryManager:
```

Comment:
because this is a new class I would say add docstrings to all public methods, maybe add some docstring to the instance variables (some but not all are now documented).

Reply:
Added